stable/spark/templates/spark-livy-hpa.yaml

```diff
@@ -0,0 +1,23 @@
+{{- if .Values.Livy.Autoscaling.Enabled }}
+apiVersion: autoscaling/v2beta1
+kind: HorizontalPodAutoscaler
+metadata:
+  labels:
+    heritage: {{ .Release.Service | quote }}
+    release: {{ .Release.Name | quote }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    component: "{{ .Release.Name }}-{{ .Values.Livy.Component }}"
```

Comment:
I really don't want to load y'all down with nitpicks here, so if this isn't the right time to do this, feel free to put it on the map for later.

The convention for the labels has changed from what it used to be.  It might be nice, since you are doing a big version jump anyway, to quickly migrate the labels over to this: https://helm.sh/docs/chart_best_practices/#standard-labels

Reply:
Thank you, yes https://github.com/helm/charts/blob/master/REVIEW_GUIDELINES.md#names-and-labels

Reply:
Thanks ! Ill put this on the map as that looks like ti would require changes to all the yamls.